flink-kubernetes/src/test/java/org/apache/flink/kubernetes/kubeclient/Fabric8FlinkKubeClientTest.java

```diff
@@ -630,4 +661,39 @@ private KubernetesConfigMap buildTestingConfigMap() {
                         .withData(data)
                         .build());
     }
+
+    private class TestingKubernetesPodCallbackHandler
+            implements FlinkKubeClient.WatchCallbackHandler<KubernetesPod> {
+
+        private final CountDownLatch eventLatch;
+
+        public TestingKubernetesPodCallbackHandler(CountDownLatch eventLatch) {
+            this.eventLatch = eventLatch;
+        }
+
+        @Override
+        public void onAdded(List<KubernetesPod> resources) {
+            this.eventLatch.countDown();
+        }
+
+        @Override
+        public void onModified(List<KubernetesPod> resources) {
+            this.eventLatch.countDown();
+        }
+
+        @Override
+        public void onDeleted(List<KubernetesPod> resources) {
+            this.eventLatch.countDown();
+        }
+
+        @Override
+        public void onError(List<KubernetesPod> resources) {
```

Comment:
ERROR event will not trigger Watcher#eventReceived, so this unit test will failed.

Maybe we can just skip this in unit test.

ref: https://github.com/fabric8io/kubernetes-client/blob/master/kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/internal/AbstractWatchManager.java#L325